lib/generators/logidze/install/templates/migration.rb.erb

```diff
@@ -9,6 +9,7 @@ class <%= @migration_class_name %> < ActiveRecord::Migration<%= ActiveRecord::VE
           BEGIN
           EXECUTE 'ALTER DATABASE ' || quote_ident(current_database()) || ' SET logidze.disabled=' || quote_literal('');
           EXECUTE 'ALTER DATABASE ' || quote_ident(current_database()) || ' SET logidze.responsible=' || quote_literal('');
```

Comment:
Do we still need this `logidze.responsible` setting?

Reply:
nope, removed the line